src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -943,8 +958,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     void TermControl::_KeyHandler(Input::KeyRoutedEventArgs const& e, const bool keyDown)
     {
-        // If the current focused element is a child element of searchbox,
-        // we do not send this event up to terminal
+        // If the current focused element is a child element of searchbox, we do
+        // not send this event up to terminal. Do this _AFTER_ the call to
+        // _TryHandleKeyBinding, so that keybindings (esp. findMatch) will still

```

Comment:
This comment still seems weird. It doesn't happen _after_ a call to TryHandleKeyBinding in any meaningful way, unless it's in a different function . . . right?

Reply:
Oh no, that comment needs to be deleted